android/build.gradle

```diff
@@ -21,12 +22,17 @@ rootProject.allprojects {
 }
 
 apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+
+sourceSets {
+  main.java.srcDirs += 'src/main/kotlin'
+}
 
 android {
-    compileSdkVersion 30
+    compileSdkVersion 31
 
     defaultConfig {
-        minSdkVersion 16
+        minSdkVersion 21
```

Comment:
Oh I guess Harry had already done all this. More than I thought, but should be good.